man/man8/zfs.8

```diff
@@ -995,15 +1017,20 @@ This property can also be referred to by its shortened column name,
 .sp
 .ne 2
 .na
-\fB\fBencryption\fR=\fBoff\fR | \fBon\fR | \fBaes-128-ccm\fR | \fBaes-192-ccm\fR | \fBaes-256-ccm\fR | \fBaes-128-gcm\fR | \fBaes-192-gcm\fR | \fBaes-256-gcm\fR
+\fB\fBencryption\fR=\fBoff\fR | \fBon\fR | \fBaes-128-ccm\fR |
+\fBaes-192-ccm\fR | \fBaes-256-ccm\fR | \fBaes-128-gcm\fR |
+\fBaes-192-gcm\fR | \fBaes-256-gcm\fR
 .ad
 .sp .6
 .RS 4n
-Controls the encryption cipher suite (block cipher, key length, and mode) used for this dataset. Requires the encryption feature to be enabled on the pool. Requires a keysource to be set at dataset creation time.
+Controls the encryption cipher suite (block cipher, key length, and mode) used
+for this dataset. Requires the encryption feature to be enabled on the pool.
+Requires a \fBkeyformat\fR to be set at dataset creation time.
 .sp
-Selecting \fBencryption\fR=\fBon\fR when creating a dataset indicates that the current default encryption suite will be selected, which is currently \fBaes-256-ccm\fR. In order to provide consistent data protection, encryption must be specified at dataset creation time and it cannot be changed afterwards.
-.sp
-The \fBencryption\fR property is inherited by child datasets by default. All inheriting datasets will be accessible when the root key is mounted. A child dataset may also set encryption locally, which will indicate that it should have a separate key and \fBkeysource\fR from the parent. Regardless, all children of an encrypted dataset must also be encrypted.
+Selecting \fBencryption\fR=\fBon\fR when creating a dataset indicates that the
+default encryption suite will be selected, which is currently \fBaes-256-ccm\fR.
+In order to provide consistent data protection, encryption must be specified at
+dataset creation time and it cannot be changed afterwards.
 .sp
 For more details and caveats about encryption see \fBzpool-features\fR.
```

Comment:
The **SEE ALSO** part of this `zfs(8)` manual page should be expanded to include `zpool-features(5)`.